### PR TITLE
TVB-2811: Display theta models in GUI

### PR DESCRIPTION
**Pull request by adrian-dordea**

### CODE REVIEW

**Comment by liadomide** [comment]
S-au testat in Phase Plane, simulări simple și la lansare de PSE aceste noi modele?

File: framework_tvb/tvb/adapters/simulator/model_forms.py [hunk 1]
```diff
@@ -517,6 +529,116 @@ def __init__(self, prefix=''):
...
+    @staticmethod
+    def get_params_configurable_in_phase_plane():
+        return ['tau', 'tau_A', 'alpha', 'I', 'Delta', 'J', 'eta', 'cr', 'cv', 'variables_of_interest']
+
+class DumontGutkinModelForm(FormWithRanges):
+
+    def __init__(self, prefix=''):
+        super(DumontGutkinModelForm, self).__init__(prefix)
+        self.I_e = ArrayField(ModelsEnum.DUMONT_GUTKIN.get_class().I_e, self)
+        self.Delta_e = ArrayField(ModelsEnum.DUMONT_GUTKIN.get_class().Delta_e, self)
```
Comment (VinczeRobert) [inline framework_tvb/tvb/adapters/simulator/model_forms.py:622]: I think this ("Delta_e") should be with lowercase letter.

File: framework_tvb/tvb/adapters/simulator/model_forms.py [hunk 2]
```diff
@@ -517,6 +529,116 @@ def __init__(self, prefix=''):
...
+        self.eta = ArrayField(ModelsEnum.GAST_SCHMIDT_KNOSCHE_SD.get_class().eta, self)
+        self.cr = ArrayField(ModelsEnum.GAST_SCHMIDT_KNOSCHE_SD.get_class().cr, self)
+        self.cv = ArrayField(ModelsEnum.GAST_SCHMIDT_KNOSCHE_SD.get_class().cv, self)
+        self.variables_of_interest = MultiSelectField(ModelsEnum.GAST_SCHMIDT_KNOSCHE_SD.get_class().variables_of_interest, self)
+
+    @staticmethod
+    def get_params_configurable_in_phase_plane():
+        return ['tau', 'tau_A', 'alpha', 'I', 'Delta', 'J', 'eta', 'cr', 'cv', 'variables_of_interest']
+
+class GastSchmidtKnosche_SFModelForm(FormWithRanges):
```
Comment (VinczeRobert) [inline framework_tvb/tvb/adapters/simulator/model_forms.py:598]: I thought it is not a good practice to have underscores in class names, but I saw it's the same way in the library.

**Review by VinczeRobert — COMMENTED** [review]
I tested running a default simulation with all these model and all of them worked, so I guess it's okay.

**Comment by i-Zaak** [comment]
Hello everyone,
 the Delta parameter has capital D as it represents Δ in the equations. As
for the underscore in the class name, that can be changed to adhere with
the convention.

Cheers,
Jan

**Comment by adrian-dordea** [comment]
I tested multiple scenarios with the new models, simple, with PSE and parameter changing, all seems ok, Phase Plane is also ok now (made a small commit to fix some problems).

**Pull request merged**